mobile/src/main/res/layout/widgetlist_buttongriditem.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    style="@style/WidgetListItemContainer">
+
+    <include layout="@layout/widgetlist_icontext_for_heavy_data" />
+
+    <TableLayout
```

Comment:
Not sure what's the best root view for the button grid.

Reply:
Either `TableLayout`, `GridLayout` or `RecyclerView` with `GridLayoutManager`. I'd probably start trying the latter, if only because I haven't worked with the former two.